packages/flutter/lib/src/material/drawer.dart

```diff
@@ -100,13 +103,29 @@ class Drawer extends StatelessWidget {
   /// {@macro flutter.widgets.child}
   final Widget child;
 
+  /// An (optional) name for the drawer used by accessibility frameworks.
+  ///
+  /// Defaults to "navigation menu" on Android, and is ignored on iOS.
+  ///
+  /// See also:
+  ///
+  ///   * [Semantics.route], the semantics flag which controls screen
+  ///     accessibility.
+  final String routeName;
+
   @override
   Widget build(BuildContext context) {
+    final String value = defaultTargetPlatform == TargetPlatform.iOS ? '' : routeName;
```

Comment:
Do we really want to make this call (iOS vs Android) here?

General note: always use a switch when dealing with enums, not ?: or if statements. That way the analyzer will catch when we add a new platform and let us know everywhere that needs to change.

Reply:
I guess it does make sense to make this call here.